src/features/creators/templates/newPackageTemplate/.github/copilot-instructions.md

```diff
@@ -0,0 +1,8 @@
+# Copilot Instructions for <package_name>
```

Comment:
See https://github.com/microsoft/vscode-python-templates/pull/4/files#diff-8263202d70318618891dd1a86fb62ccab9567201cc456e0085703a8c400b95f6 for additional items we can add to the instructions file

Reply:
my question here is if I should add all these general rules or if that wont feel applicable to the action the user is taking of adding the project? cc @karthiknadig